uber/templates/regform.html

```diff
@@ -297,15 +297,19 @@
         {% endif %}
     {% endif %}
 
+{%- set readonly_badge_name =
+    (c.AFTER_PRINTED_BADGE_DEADLINE and attendee.badge_type in c.PREASSIGNED_BADGE_TYPES)
+    or (c.AFTER_SUPPORTER_DEADLINE and attendee.badge_type not in c.PREASSIGNED_BADGE_TYPES) -%}
```

Comment:
Should these two clauses use an `and` or an `or`?

Reply:
Also, is this doing what we want?

Reply:
Oh, we actually have a function just for this! I guess it... never got ported to the template? 😰  It's here: https://github.com/magfest/ubersystem/blob/master/uber/config.py#L141

It's up to you how to want to implement that (a function call might work?), but the basic idea behind it is that we wanted to actually let people edit their Supporter badges _after_ their Staff badges, because customized Staff badges get ordered from a manufacturer but Supporter badges are often hand-made much closer to the event!

Reply:
Nice! Thank you for pointing me in the right direction!